
# HG changeset patch
# User Darren Salt <linux@youmustbejoking.demon.co.uk>
# Date 1238800102 -3600
# Node ID afe0ffd88733cde367d617129f9b4bff157e9d03
# Parent dc900e3ef758c3b7e2432c99f7973d89d22bfb0f
Correct invalid MIME info in the MOD demuxer.

--- a/src/demuxers/demux_mod.c	Fri Apr 03 19:31:34 2009 +0100
+++ b/src/demuxers/demux_mod.c	Sat Apr 04 00:08:22 2009 +0100
@@ -387,11 +387,11 @@
          "audio/x-s3m: s3m: ScreamTracker 3 Module;"
          "audio/s3m: s3m: ScreamTracker 3 Module;"
          "application/playerpro: 669: 669 Tracker Module;"
-         "application/adrift; amf: ADRIFT Module File;"
-         "audio/med; med; Amiga MED/OctaMED Tracker Module Sound File;"
-         "audio/x-amf; amf: ADRIFT Module File;"
-         "audio/x-xm; xm: FastTracker II Audio;"
-         "audio/xm; xm: FastTracker II Audio;";
+         "application/adrift: amf: ADRIFT Module File;"
+         "audio/med: med: Amiga MED/OctaMED Tracker Module Sound File;"
+         "audio/x-amf: amf: ADRIFT Module File;"
+         "audio/x-xm: xm: FastTracker II Audio;"
+         "audio/xm: xm: FastTracker II Audio;";
 }
 
 static void class_dispose (demux_class_t *this_gen) {

